api/src/types/s2-api.ts

```diff
@@ -13,21 +13,52 @@ interface S2ApiSuccess {
   status: 200;
 }
 
-export interface S2ApiPaper {
-  abstract: string;
-  arxivId: string | null;
-  authors: S2ApiAuthor[];
-  doi: string;
-  title: string;
+export interface S2Api {
+  paper: S2ApiPaper;
+}
+
+interface S2ApiPaper {
+  id: string;
+  title: S2ApiTitle;
+  paperAbstract: S2ApiAbstract;
+  authors: S2ApiAuthor[][];
+  year: S2ApiYear;
+  venue: S2ApiVenue;
+  primaryPaperLink: S2ApiPaperLink;
+  doiInfo: S2ApiDoi;
+  citationStats: S2ApiCitationStats;
+}
+
+export interface S2ApiTitle {
+  text: string;
+}
+
+interface S2ApiAbstract {
+  text: string;
+}
+
+interface S2ApiYear {
+  text: string;
+}
+
+interface S2ApiVenue {
+  text: string;
+}
+
+interface S2ApiPaperLink {
   url: string;
-  venue: string;
-  year: string;
-  influentialCitationCount?: number;
-  citationVelocity?: number;
+}
+
+interface S2ApiDoi {
+  doi: string;
+}
+
+interface S2ApiCitationStats {
+  citationVelocity: number;
+  numKeyCitations: number;
 }
 
 interface S2ApiAuthor {
-  authorId: string;
+  ids: string;
```

Comment:
Should this just be `id` if it's a single ID?

Reply:
![image](https://user-images.githubusercontent.com/52334652/82385923-16129380-99e8-11ea-9070-80d8e3c178ca.png)

this is what the interface has :/